auto-value-parcel/src/main/java/com/ryanharter/auto/value/parcel/Parcelables.java

```diff
@@ -42,6 +43,7 @@
   private static final TypeName SIZE = ClassName.get("android.util", "Size");
   private static final TypeName SIZEF = ClassName.get("android.util", "SizeF");
   private static final TypeName TEXTUTILS = ClassName.get("android.text", "TextUtils");
+  private static final TypeName ENUM = ClassName.get(Enum.class);
 
   private static final Set<TypeName> VALID_TYPES = ImmutableSet.of(STRING, MAP, LIST, BOOLEANARRAY,
```

Comment:
Is `ENUM` not in `VALID_TYPES` so that you can prioritize `Parcelable` over the `Enum` case?